packages/aws-rfdk/docs/upgrade/upgrading-0.37.md

```diff
@@ -0,0 +1,41 @@
+# Upgrading to RFDK v0.37.x or Newer
+
+Starting in RFDK v0.37.0, the default for TLS between the render queue and its clients, which is configured using the `RenderQueueExternalTLSProps` interface that the `RenderQueue` construct takes as a part of its constructor props, is now set to be enabled.
+
+## Upgrading Farms Already Using TLS
+
+If you are already setting fields on the `RenderQueueExternalTLSProps` for the Render Queue, no action is required. Redeploying your render farm after upgrading your version of RFDK should have no effect.
+
+## Upgrading Farms Not Using TLS
```

Comment:
While you have the options documented in this file, perhaps it would help make things clearer if we added a paragraph at this level (just below this heading) outlining the two upgrade paths with links to each sub-section:

1.  Migrating to TLS (recommended)
2.  Preserving plain HTTP

Do you think it would help?